### ISSUE TITLE
Text selection is not working in "Add transaction" dialog

### ISSUE THREAD

**Issue by xuhcc** [beancount/fava#1015]
After a recent change (ad3b3435eef0df71d0224ba2676926f5aa548854) to "Add transaction" dialog, it is no longer possible to select text in postings with a mouse. The whole posting is entering a drag-n-drop mode instead.

As an alternative to making the whole posting draggable, I would suggest adding a small 'drag' button (e.g. [↕](https://unicode-table.com/en/sets/arrows-symbols/)) on the left side of the posting, near the 'remove' button:

![Screen Shot 2019-11-20 at 19 11 06](https://user-images.githubusercontent.com/1660460/69255914-abfc6b80-0bc9-11ea-9214-0a4865cb70c2.png)

**Comment by xuhcc** [comment]
Text selection still not working in v1.12. Please reopen.

**Comment by yagebu** [comment]
Mhh, this is tricky.

PS: Can't you reopen the issue yourself?

**Comment by xuhcc** [comment]
I just thought that it's up to you to decide whether to reopen it or not :)

**Comment by xuhcc** [comment]
It seems that a problem is limited to Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1189486
In Chromium I can select the text.

**Comment by yagebu** [comment]
> I just thought that it's up to you to decide whether to reopen it or not :)

:) - you're always free to reopen it yourself though - if we disagree, I could still close it again :D I might more easily forget about a closed issue.

> It seems that a problem is limited to Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1189486

Yeah, I saw that one too. I thought it also had some minor problems in Chromium though.

In any case, I think I found a more complete fix - on mouseover, toggle the draggable state depending on whether we are over one of the inputs or not.
